lib/exec.js

```diff
@@ -197,7 +197,7 @@ function _execUrl(context, configVars) {
     return url.parse(urlString);
   } else {
     if (process.env.HEROKU_EXEC_URL === undefined) {
```

Comment:
should this be `if (process.env.HEROKU_EXEC_URL)`?

Reply:
i don't know, should it? :)

Reply:
TIL:

```
>> ("") ? console.log("true") : console.log("false")
false
```